xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
I begin to understand:

double cachetotal = DVD_SEC_TO_TIME(m_dvdAudio.GetCacheTotal());
Is the time wich is cached for decoding? Is this right? If so I have to search more in this direction.